tensorboard/plugin_util_test.py

```diff
@@ -108,6 +108,14 @@ def test_unicode_strings_passed_through(self):
     self._test(s,
                u'<blockquote>\n<p>Look\u2014some UTF-8!</p>\n</blockquote>')
 
+  def test_null_bytes_stripped_before_markdown_processing(self):
+    # If this function is mistakenly called with UTF-16 or UTF-32 encoded text,
+    # there will probably be a bunch of null bytes. These would be stripped by
+    # the sanitizer no matter what, but make sure we remove them before markdown
+    # interpretation to avoid affecting output (e.g. "_with_" gets italicized).
+    s = u'word_with_underscores'.encode('utf-16')[2:]  # Strip byte-order mark
```

Comment:
If the 2 chars are not stripped, do we get a `UnicodeDecodeError`? Is that why we strip them?

Reply:
Yeah, that's why.  I realized that using "-le" or "-be" suffixes on the encoding to specify endianness will avoid writing out a byte order mark in the first place though which is significantly cleaner, so now I've updated this to just use utf-32-le since that's the exact encoding Numpy uses.